charts/vllm/templates/deployment.yaml

```diff
@@ -39,11 +39,15 @@ spec:
               containerPort: {{ .Values.port }}
               protocol: TCP
           startupProbe:
-            httpPost:
-              path: /v1/completions
-              port: {{ .Values.port }}
-              body: '{"prompt": "Hi there", "max_tokens": 5, "model": "{{ include "vllm.usedModelName" . }}"}'
-            initialDelaySeconds: 10
+            exec:
+              command:
+              - sh
+              - -c
+              - |
+                curl -X POST "http://localhost:{{ .Values.port }}/v1/completions" \
```

Comment:
This will return a 0 status code on non-200 HTTP responses I think.

Reply:
I tested it and returns a 0 status code in the success case